### PR TITLE
Added button component, 

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added button component, moved utility classes to a new css file, styled button at landing page using button component.

### CODE REVIEW

File: html/docs.html [hunk 1]
```diff
@@ -196,6 +196,42 @@ <h2 class="medium-heading docs-heading" id="badge">
                     </code>
                   </figure>
                 </div>
+
+                <h2 class="medium-heading docs-heading" id="button">
+                  Button
+              </h2>
+                <p class="mx-2 fs-4 mt-3 lh-25p">
+                Buttons are call to action.
```
Comment (saadahmadhere) [inline html/docs.html]: this line should be indented properly.

**Review by saadahmadhere — APPROVED** [review]
Code looks nice and clean. Check the indentation issue. Good to merge after that.

**Review by hinanshisuthar — COMMENTED** [review]
Use any code formatter such as prettify or beautier. Everything else looks pretty good to merge.

File: html/docs.html (hunk 1 above)
Comment (nareshpoonia) [reply html/docs.html]: fixed the indentation

**Comment by nareshpoonia** [comment]
Fixed the indentation

**Review by stutisk — APPROVED** [review]
Good to merge

**Pull request merged**